lib/polyfill/storage_estimate.js

```diff
@@ -0,0 +1,54 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.polyfill.StorageEstimate');
+
+goog.require('shaka.polyfill');
+
+
+/**
+ * @summary A polyfill to provide navigator.storage.estimate in old
+ * webkit browsers.
+ */
+shaka.polyfill.StorageEstimate = class {
+  /**
+   * Install the polyfill if needed.
+   */
+  static install() {
+    if ('storage' in navigator && 'estimate' in navigator.storage) {
```

Comment:
Unless being `null` is expected and supported, it would be more consistent to just check the fields directly.

```js
if (navigator.storage && navigator.storage.estimate) {
```

Reply:
Done!